examples/panic.rs

```diff
@@ -87,6 +87,7 @@ fn reset_terminal() -> Result<()> {
 }
 
 /// Runs the TUI loop.
+#[allow(clippy::no_effect)]
```

Comment:
This seems unnecessary. Other than that, disregard my previous comment, until crossterm merges the opt out default, it makes sense to have this explicitly handled.

Reply:
Ah yes that was a result of not remembering to undo it after removing the powershell hack, which is no longer needed assuming the crossterm PR will be merged.

The press events check doesn't necessarily need to be removed later, as making use of the release events could have some uses.